atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleOverflowWriteTable.java

```diff
@@ -398,18 +408,22 @@ private void deleteAllTimestampsOverflow(String overflowTable, List<Object[]> ar
         String shortTableName = oraclePrefixedTableNames.get(tableRef, conns);
         conns.get()
                 .updateManyUnregisteredQuery(
-                        " /* DELETE_ALL_TS_OVERFLOW (" + overflowTable + ") */ "
-                                + " DELETE /*+ INDEX(m " + PrimaryKeyConstraintNames.get(overflowTable) + ") */ "
-                                + "   FROM " + overflowTable + " m "
-                                + "  WHERE m.id IN (SELECT /*+ INDEX(i " + PrimaryKeyConstraintNames.get(shortTableName)
-                                + ") */ "
-                                + "                        i.overflow "
-                                + "                   FROM " + shortTableName + " i "
-                                + "                  WHERE i.row_name = ? "
-                                + "                    AND i.col_name = ? "
-                                + "                    AND i.ts >= ? "
-                                + "                    AND i.ts <= ? "
-                                + "                    AND i.overflow IS NOT NULL)",
+                        " /* DELETE_ONE_OVERFLOW (" + overflowTable + ") */"
```

Comment:
surely this should remain `DELETE_ALL_TS_OVERFLOW`?

Reply:
should be fixed now